app/java/makefile

```diff
@@ -12,23 +11,29 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
 
+# If you're on Windows, use mvnw.cmd instead of ./mvnw.
+
+# Install dependencies
 install:
-	echo "noop"
+	./mvnw dependency:resolve
+
+# Build the web app and processing job
+build:
+	./mvnw package
```

Comment:
Heads-up: I introduced `make build` (since Java is a compiled language).
Let me know if I should use a different word.

Reply:
When is `build` needed? If it's needed before each of `lint` and `test`, the package command should be part of `install`.

makefile commands are mostly used in the github action defined in unit-test.yaml, see that file for parity considerations.

Reply:
> When is `build` needed?

Good question.
We do not need to manually run `make build` before `make lint` and `make test`.
That is, `mvn` will run `mvn package` (`make build`) automatically (during the invocation of `make lint` and `make test`).

I included it for two reasons:
1. for completeness (if anyone just wants to create the build artifacts)
2. for https://screenshot.googleplex.com/M9LeH7fHMb5k6oJ.png and https://github.com/GoogleCloudPlatform/terraform-cloud-client-api/pull/32/commits/1c4ac4da60e953bd99b744bd52f352c7709d8ac6

I didn't reusing the term `install` (`make install`) because `mvn package` — in additional to installing deps — compiles source code and builds the JAR files too.

I also used `build` because of [go/samples-repo-interface](https://docs.google.com/document/d/1IamHopFETGlzyaYKDcU-Q82-7FBXYXZt2nsCdKC-4AY/edit?tab=t.0#heading=h.wbe2do2rt4ae), which provides a standard set of make commands.

That said, I am open either removing `make build` or renaming it. Your call.